src/main/java/com/spotify/sparkey/Util.java

```diff
@@ -266,41 +267,46 @@ static void nonThrowingClose(Closeable stream) {
     }
   }
 
-  static boolean renameFile(File srcFile, File destFile) throws FileNotFoundException {
+  static void renameFile(File srcFile, File destFile) throws IOException {
     if (!srcFile.exists()) {
       throw new FileNotFoundException(srcFile.getPath());
     }
     if (srcFile.equals(destFile)) {
```

Comment:
Equals for files is not well defined, this can be a false positive

Reply:
Should be fine for this case